management-core/src/main/java/com/ns/task/ManagementApplicationCore.java

```diff
@@ -0,0 +1,11 @@
+package com.ns.task;
```

Comment:
What is this class for?

Reply:
This is because in order to run the application as spring boot I need an entry point so this will be, maybe I change the CoreConfig class bean`s into this one, what do you thing about?